programs/application_factory/src/lib.rs

```diff
@@ -2,14 +2,50 @@ use anchor_lang::prelude::*;
 
 declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
 
+const JOB_APPLICATION_SEED: &'static [u8] = b"job_application";
+
 #[program]
 pub mod application_factory {
     use super::*;
 
-    pub fn initialize(ctx: Context<Initialize>) -> Result<()> {
+    pub fn initialize(ctx: Context<Initialize>, job_ad_id: String) -> Result<()> {
+
+        let details = &mut ctx.accounts.base_account;
+
+        details.status = false,
```

Comment:
Can you create a method for this object that's call something like "initialize" or "reset" and then we avoid having so many lines in the instruction by calling it directly?

Reply:
ok ill check it out